irohad/ametsuchi/storage.hpp

```diff
@@ -10,11 +10,13 @@
 
 #include <rxcpp/rx.hpp>
 #include "ametsuchi/block_query_factory.hpp"
+#include "ametsuchi/commit_result.hpp"
 #include "ametsuchi/mutable_factory.hpp"
 #include "ametsuchi/peer_query_factory.hpp"
 #include "ametsuchi/query_executor_factory.hpp"
 #include "ametsuchi/temporary_factory.hpp"
 #include "common/result.hpp"
+#include "interfaces/common_objects/range_types.hpp"
```

Comment:
Do we need this headers right here?

Reply:
thank you, i missed these after rebasing commits from initial PR